imagenet_classes.csv

```diff
@@ -0,0 +1,1000 @@
+0tench, Tinca tinca
```

Comment:
In between the index (0) and label (tench, Tinca tinca) there is the non-printable character "\001", that I use to split the field with in the Python code.